db/seeds.d/010-calibrate_bcrypt_cost.rb

```diff
@@ -0,0 +1,11 @@
+# calibrate default cost to be at least 5ms (performance-wise safe)
+new_cost = BCrypt::Engine.calibrate(5)
+
+# keep minimum reasonable cost in case CPU was busy
+new_cost = 4 if new_cost < 4
```

Comment:
Same here with 4.

Reply:
Same concern - you want me to expose now three numbers, that's a challenge to explain for advanced users and impossible for noobs - they will have no idea how to set this and they will set it wrong.

Constants are for good, please don't make it dogma to expose them all that's not possible. This one is a security check to avoid setting cost to 1 in case CPU was busy during the short test. Cost values 1-4 can be calculated under 1 ms. I can however remove this check, having CPU so slow that it would end up being cost 1 is highly unlikely.